starlite/utils/types.py

```diff
@@ -35,3 +50,16 @@ def annotation_is_iterable_of_type(
     ):
         return args[0] is type_value or isinstance(args[0], type_value) or is_class_and_subclass(args[0], type_value)
     return False
+
+
+def make_non_optional_union(annotation: Optional[UnionT]) -> UnionT:
+    """Make a `Union` type that excludes `NoneType`.
```

Comment:
```suggestion
    """Make a :data:`Union <typing.Union>` type that excludes ``NoneType``.
```

Reply:
thanks for these comments, I'll have to brush up on my ReST!